src/Illuminate/Database/Schema/Grammars/SQLiteGrammar.php

```diff
@@ -28,6 +26,23 @@ class SQLiteGrammar extends Grammar
      */
     protected $serials = ['bigInteger', 'integer', 'mediumInteger', 'smallInteger', 'tinyInteger'];
 
+    /**
+     * Get the commands to be compiled on the alter command.
+     *
+     * @param  \Illuminate\Database\Connection  $connection
+     * @return array
+     */
+    public function getAlterCommands(Connection $connection)
+    {
+        $alterCommands = ['change', 'primary', 'dropPrimary', 'foreign', 'dropForeign'];
+
+        if (version_compare($connection->getServerVersion(), '3.35', '<')) {
```

Comment:
@hafezdivandari shouldn't the operator be ">=", here? If the goal is to make sure SQLite >= 3.35 uses the `dropColumn` command instead of recreating the table?

Reply:
No, this method prepares an array of commands that are not supported by SQLite, and we have to recreate the table as a workaround for them. So as dropping column is not supported on <3.35 we add `dropColumn` command to the list.